docs/overview.md

```diff
@@ -8,46 +8,85 @@ At the begging I expect, that you already have:
-        'username': 'default',
-        'password': ''
+* if you already have a celery work flow:
+    ```python
+    # django-clickhouse library setup
+    CLICKHOUSE_DATABASES = {
+        # Connection name to refer in using(...) method 
+        'default': {
+            'db_name': 'test',
+            'username': 'default',
+            'password': ''
+        }
     }
-}
-CLICKHOUSE_REDIS_CONFIG = {
-    'host': '127.0.0.1',
-    'port': 6379,
-    'db': 8,
-    'socket_timeout': 10
-}
-CLICKHOUSE_CELERY_QUEUE = 'clickhouse'
-
-# If you have no any celerybeat tasks, define a new dictionary
-# More info: http://docs.celeryproject.org/en/v2.3.3/userguide/periodic-tasks.html
-from datetime import timedelta
-CELERYBEAT_SCHEDULE = {
-    'clickhouse_auto_sync': {
-        'task': 'django_clickhouse.tasks.clickhouse_auto_sync',
-        'schedule': timedelta(seconds=2),  # Every 2 seconds
-        'options': {'expires': 1, 'queue': CLICKHOUSE_CELERY_QUEUE}
+    CLICKHOUSE_REDIS_CONFIG = {
+        'host': '127.0.0.1',
+        'port': 6379,
+        'db': 8,
+        'socket_timeout': 10
     }
-}
-```
+    CLICKHOUSE_CELERY_QUEUE = 'clickhouse'
+
+    # If you have no any celerybeat tasks, define a new dictionary
+    # More info: http://docs.celeryproject.org/en/v2.3.3/userguide/periodic-tasks.html
+    from datetime import timedelta
+    CELERYBEAT_SCHEDULE = {
+        'clickhouse_auto_sync': {
+            'task': 'django_clickhouse.tasks.clickhouse_auto_sync',
+            'schedule': timedelta(seconds=2),  # Every 2 seconds
+            'options': {'expires': 1, 'queue': CLICKHOUSE_CELERY_QUEUE}
+        }
+    }
+    ```
+* if you don't have a celery workflow:
+
+    create a `celery.py` file in `mysite/mysite/celery.py`:
+    ```python
+    import os
+    from datetime import timedelta
+    from celery import Celery
+
+    # set the default Django settings module for the 'celery' program.
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "mysite.settings")
+
+    app = Celery("podafarini")
+
+    # Using a string here means the worker doesn't have to serialize
+    # the configuration object to child processes.
+    # - namespace='CELERY' means all celery-related configuration keys
+    #   should have a `CELERY_` prefix.
+    app.config_from_object("django.conf:settings", namespace="CELERY")
+    # Load task modules from all registered Django app configs.
+    app.autodiscover_tasks()
+    app.conf.beat_schedule = {
+        "clickhouse_auto_sync": {
+            "task": "django_clickhouse.tasks.clickhouse_auto_sync",
+            "schedule": timedelta(seconds=2),  # Every 2 seconds
+            "options": {"expires": 1, "queue": "clickhouse"},
+        }
+    }
+    ```
+    `mysite/mysite/__init__.py`:
+    ```python
+    # This will make sure the app is always imported when
+    # Django starts so that shared_task will use this app.
+    from .celery import app as celery_app
+
+    __all__ = ("celery_app",)
+
+    ```
```

Comment:
Instead of this, I'd better add [link to celery installation process](https://docs.celeryproject.org/en/stable/django/first-steps-with-django.html):
1. It is written better there
2. Celery setup can change any time and we would not change docs here